pkg/util/log/log.go

```diff
@@ -383,6 +384,13 @@ func Warnf(format string, params ...interface{}) error {
 func Errorf(format string, params ...interface{}) error {
 	if logger != nil && logger.inner != nil && logger.shouldLog(seelog.ErrorLvl) {
 		return logger.errorf(format, params...)
+	} else if logger == nil || logger.inner == nil {
+		// We're currently trying to log an error before initializing
+		// the log module. This meant a early error while starting the
+		// agent. We should not silence such error.
+		err := fmt.Errorf(format, params...)
+		fmt.Fprintf(os.Stderr, "Error: %s", err)
```

Comment:
Maybe `log` would be better fit than `fmt`?

Reply:
:+1:

Reply:
The package `log` has a default logger
> That logger writes to standard error and prints the date and time of each logged message